src/pybind/mgr/dashboard/README.rst

```diff
@@ -18,7 +18,7 @@ documentation
 for details on how to accomplish this.
 
 Finally, you need to build the dashboard frontend code. See the file
-``HACKING.rst`` in this directory for instructions on setting up the necessary
+:ref:`hacking` for instructions on setting up the necessary
```

Comment:
That reference will likely not work, as the `README.rst` file is not part of the official documentation. I'd suggest to replace this with an URL to the documentation website instead, or a reference to `doc/dev/developer_guide/HACKING.rst` in the git repository.

Reply:
However, as part of the move into the official Ceph documentation, I'd suggest to use a more intuitive file name other than `HACKING.rst` - how about `dashboard-developer-documentation.rst` instead?

Reply:
or just `dashboard.rst` as it's located under the `developer_guide` directory already.